client/my-sites/customer-home/cards/tasks/black-friday-2024/index.jsx

```diff
@@ -0,0 +1,31 @@
+import { useTranslate } from 'i18n-calypso';
+import { useSelector } from 'react-redux';
+import blackFriday2024Illustration from 'calypso/assets/images/customer-home/illustration--black-friday-2024.jpg';
+import { TASK_BLACK_FRIDAY_2024 } from 'calypso/my-sites/customer-home/cards/constants';
+import Task from 'calypso/my-sites/customer-home/cards/tasks/task';
+import { getSelectedSiteSlug } from 'calypso/state/ui/selectors';
+
+import './style.scss';
+
+const Affiliates = () => {
```

Comment:
This looks like a typo. Maybe somethjing like `BlackFriday2024`?

Reply:
Thanks for catching that!
Fixed.

Reply:
Can we pls update the headline to "Our Biggest Sale..."

And also use this promo visual for the image: nNq0GqvlzdQ5i3OgusUon9-fi-249_65735 

Thanks!

Reply:
@jrdnhllr Headline and promo visual have been updated.